src/App.tsx

```diff
@@ -1,8 +1,115 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
+import {
+  getComments, getPosts, getUsers, Posts,
+} from './helpers/api';
+import { LoadingSpinner } from './loading-spinner';
 
-const App = () => (
-  <h1>Dynamic list of posts</h1>
-);
+const App = () => {
+  const [post, setPost] = useState<Posts[]>([]);
```

Comment:
it's an array, right? but name is `post`

Reply:
fixed